nzedb/Music.php

```diff
@@ -442,11 +449,12 @@ public function update($id, $title, $asin, $url, $salesrank, $artist, $publisher
 	}
 
 	/**
-	 * @param             $title
-	 * @param             $year
-	 * @param object|null $amazdata
+	 * @param      $title
+	 * @param      $year
+	 * @param null $amazdata
```

Comment:
This is wrong. null is never a type on it's own. Please restore the original line.